src/cmd/services/m3coordinator/downsample/downsampler.go

```diff
@@ -129,7 +129,8 @@ func defaultMetricsAppenderOptions(opts DownsamplerOptions, agg agg) metricsAppe
 	}
 	scope := opts.InstrumentOptions.MetricsScope().SubScope("metrics_appender")
 	metrics := metricsAppenderMetrics{
-		processedCount: scope.Counter("processed_count"),
+		processedCountNonRollup: scope.Tagged(map[string]string{"aggType": "non_rollup"}).Counter("processed_count"),
+		processedCountRollup:    scope.Tagged(map[string]string{"aggType": "rollup"}).Counter("processed_count"),
```

Comment:
nit: For tag names we use either `-` dash separated or `_` underscore separated for consistency.